web/src/components/Editor/typesResolve/index.ts

```diff
@@ -3,12 +3,15 @@ import * as monaco from "monaco-editor";
 
 import { globalDeclare } from "./globals";
 
+import useGlobalStore from "@/pages/globalStore";
+
 async function loadPackageTypings(packageName: string) {
+  const { currentApp } = useGlobalStore.getState();
+  const url = `//${currentApp?.gateway.status.appRoute.domain}`;
```

Comment:
in 0.8x, the http protocol is came from 'store.state.app.app_deploy_url_schema || 'http'',  but there is no such field in new version.